iis-workflows/iis-workflows-core-examples/src/test/java/eu/dnetlib/iis/core/examples/spark/PersonIdExtractorWorkflowTest.java

```diff
@@ -0,0 +1,34 @@
+package eu.dnetlib.iis.core.examples.spark;
+
+import org.junit.Test;
+
+import eu.dnetlib.iis.core.AbstractOozieWorkflowTestCase;
+import eu.dnetlib.iis.core.OozieWorkflowTestConfiguration;
+import eu.dnetlib.iis.core.TestsIOUtils;
+import eu.dnetlib.iis.core.WorkflowTestResult;
+
+/**
+ * 
+ * @author madryk
+ *
+ */
+public class PersonIdExtractorWorkflowTest extends AbstractOozieWorkflowTestCase {
```

Comment:
Surefire in version that we are using will inherit category from base class. I checked it.
http://stackoverflow.com/questions/27248795/why-does-junit-ignore-category-annotations-on-base-classes